src/utils.js

```diff
@@ -4,15 +4,19 @@ import styled from 'styled-components';
 export const themeGet = (lookup, fallback) => ({ theme } = {}) => get(theme, lookup, fallback);
 
 export const getComponentVariant = (theme, componentName, variant) => {
-  const config = themeGet(`components.${componentName}.variants.${variant}`, theme.variants[componentName][variant])({
+  const config = themeGet(`variants.${componentName}.${variant}`, theme.variants[componentName][variant])({
     theme,
   });
+
   if (!config) throw new Error(`Molekule: "${variant}" variant not found in theme...`);
   return config;
 };
 
 export const getComponentStyle = componentName => themeGet(`components.${componentName}.style`);
 
+export const getComponentVariantStyles = (componentName, variant) =>
+  themeGet(`components.${componentName}.variants.${variant}.style`);
```

Comment:
This will return a function instead of a value because `themeGet` returns a function that accepts `theme` as an argument.

Reply:
This seems to be the way it has been used here https://github.com/sappira-inc/molekule/blob/0656ca13a18efe5b70ebbced48f3848572fe321f/src/utils.js#L15 ? LMK